### ISSUE TITLE
Homebrew formula?

### ISSUE THREAD

**Issue by rattrayalex** [kurtbuilds/dye#1]
Currently `brew install dye` returns "No available formula". Thoughts on listing this project there so folks can install without cargo?

**Comment by kurtbuilds** [comment]
Yeah, I'd love to, I've just never created a homebrew package. If you have a pain-free tutorial (or even better, a PR 😉), happy to get this published to homebrew.

**Comment by rattrayalex** [comment]
Heh, from [their docs](https://docs.brew.sh/Acceptable-Formulae) it looks like it's best if I do this:

> We frown on authors submitting their own work unless it is very popular.

But sadly I didn't make much progress before I hit my timebox. I suspect that dye may not have enough usage yet to be merged even if a user submits it in any case.

Looks like I could use [`homebrew-rust-crate`](https://github.com/sl4m/homebrew-rust-crate) to easily install this with:

```bash
brew install sl4m/rust-crate/brew-crate
brew crate dye
```

However, that didn't work out of the box for me.
